redbox/models/settings.py

```diff
@@ -125,8 +125,12 @@ class Settings(BaseSettings):
 
     openai_api_version: str = "2023-12-01-preview"
     azure_openai_model: str = "azure/gpt-35-turbo-16k"
+    azure_embedding_model: str = "text-embedding-ada-002"
```

Comment:
Just to double check, we only have 350K TPM approved for ada, but we do have 700K TPM for text-embedding-3-large which is more performant. Do we want to swap out now?

Reply:
Clarified that this is happening in later PR